doc/whats_new.rst

```diff
@@ -794,6 +794,11 @@ Miscellaneous
   :mod:`utils` have been removed or deprecated accordingly.
   :issue:`8854` and :issue:`8874` by :user:`Naoya Kanai <naoyak>`
 
+- The ``store_covariances`` and ``covariances_`` parameters of
+  :class:`discriminant_analysis.QuadraticDiscriminantAnalysis`
+  has been renamed to ``store_covariance`` and ``covariance_``,they will be
```

Comment:
It might be helpful to give some motivation for this API change:

    has been renamed to ``store_covariance`` and ``covariance_`` to be
    consistent with the corresponding parameter names of the LinearDiscriminantAnalysis class.